docs/source/tutorials/e2e_flow.rst

```diff
@@ -330,7 +332,7 @@ these parameters.
 
 .. code-block:: text
 
-    $ tune run generate --config ./custom_generation_config.yaml prompt="tell me a joke. "
+    $ tune run generate --config ./custom_generation_config.yaml
```

Comment:
could you try running this?
`tune run generate --config ./custom_generation_config.yaml prompt.user="Tell me a joke. "`

if that works fine, would prefer including this as an example just to be explicit